promptsource/templates/wikiann/en/templates.yaml

```diff
@@ -0,0 +1,189 @@
+dataset: wikiann
+subset: en
+templates:
+  41b109ce-7504-4263-b455-315fa7a7f3c2: !Template
+    answer_choices: Yes ||| No
+    id: 41b109ce-7504-4263-b455-315fa7a7f3c2
+    jinja: 'Given, "{{''  ''.join(tokens)}}", is {{spans[0].split('':'')[1]}} a location?
+
+
+      |||
+
+      {% if spans[0][:3] == ''LOC'' %}
+
+      Yes
+
+      {% else %}
+
+      No
+
+      {% endif %}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - AUC
+      - COQA F1
+      - Other
+      - Accuracy
+      original_task: true
+    name: is_location
+    reference: ''
+  4b14afc2-38bd-4f55-9273-58ab2f996157: !Template
+    answer_choices: Yes ||| No
+    id: 4b14afc2-38bd-4f55-9273-58ab2f996157
+    jinja: 'Given, "{{''  ''.join(tokens)}}", is {{spans[0].split('':'')[1]}} an organization?
+
+
+      |||
+
+      {% if spans[0][:3] == ''ORG'' %}
+
+      Yes
+
+      {% else %}
+
+      No
+
+      {% endif %}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - AUC
+      - Accuracy
+      - COQA F1
+      - Other
+      original_task: true
+    name: is_organization
+    reference: ''
+  53bd1751-f0e6-4ca8-84d4-80c1c09a230d: !Template
+    answer_choices: True ||| False
+    id: 53bd1751-f0e6-4ca8-84d4-80c1c09a230d
+    jinja: 'Given the following information:
```

Comment:
As opposed to "Given", or "Given the following information:", maybe it's more natural to say "in the following sentence…"?